.github/workflows/container.yml

```diff
@@ -0,0 +1,56 @@
+name: Container
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  docker:
+    name: Docker
+    runs-on: ubuntu-latest
+    steps:
+      - id: checkout
+        name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - id: setup
+        name: Setup Docker Toolchain
+        uses: docker/setup-buildx-action@v2
+
+      - id: build
+        name: Build Image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          load: true
+          push: false
+          tags: index-backend:local
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - id: inspect
+        name: Inspect Inspect
+        run: docker image inspect index-backend:local
```

Comment:
Hi @da2ce7 when you generate the local docker image here, is that image available when you run:

```
- id: test
  name: Run Integration Tests
  run: ./docker/bin/run-e2e-tests.sh
```

here: https://github.com/torrust/torrust-index-backend/pull/248/files#diff-cf3dd985ec02b2c727885d15dd9a5eca1e2f1ed708cb81406d99f57099aed253R126

I see it's not: https://github.com/torrust/torrust-index-backend/actions/runs/5754190543/job/15598977722?pr=248#step:3:251

You need to save an artefact or push it to a registry.